orchagent/aclorch.cpp

```diff
@@ -1898,18 +2024,17 @@ sai_status_t AclOrch::bindAclTable(sai_object_id_t table_oid, AclTable &aclTable
     sai_status_t status = SAI_STATUS_SUCCESS;
 
     SWSS_LOG_INFO("%s table %s to ports", bind ? "Bind" : "Unbind", aclTable.id.c_str());
+    
+    if (aclTable.portSet.empty())
+    {
+        SWSS_LOG_ERROR("Port list is not configured for %s table", aclTable.id.c_str());
+        return SAI_STATUS_FAILURE;
+    }
 
     if (aclTable.ports.empty())
     {
-        if (bind)
-        {
-            SWSS_LOG_ERROR("Port list is not configured for %s table", aclTable.id.c_str());
-            return SAI_STATUS_FAILURE;
-        }
-        else
-        {
-            return SAI_STATUS_SUCCESS;
-        }
+        SWSS_LOG_WARN("Binding port list is empty for %s table", aclTable.id.c_str());
```

Comment:
It could be a normal flow to delete aclTable which is not _yet_ bind to any port list. We wouldn't want to log warning if it is in the delete flow.

Reply:
condition check added.